apps/server-web/src/renderer/pages/setup/Landing.tsx

```diff
@@ -0,0 +1,46 @@
+import { EverTeamsLogo } from '../../components/svgs/index';
+type props = {
+  nextAction: () => void;
+};
+const Landing = (props: props) => {
+  return (
+    <div className="w-full">
+      <div className="mb-6 ml-10">
+        <select className="dark:bg-gray-800 text-gray-700 dark:text-white py-2 px-4 rounded-lg border-2 dark:border-none">
+          <option>English</option>
+          {/* Add other language options here */}
+        </select>
+      </div>
+
+      <div className="mb-8 w-full text-center">
+        <div className="text-center m-auto mt-8 mb-8 w-48 items-center">
+          <EverTeamsLogo />
+        </div>
+        <h1 className="text-2xl font-bold mb-4 dark:text-gray-400 text-gray-700">
+          Welcome to Ever® Teams - Open-Source Business Management Platform
```

Comment:
@syns2191 should be usage of translation files here everywhere in this file really!